### PR TITLE
travel.navitime.com

### LINKED ISSUE #109544: travel.navitime.com
### Issue URL (Ads)
 [https://travel.navitime.com/ja/area/jp/guide/NTJmat1113](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftravel.navitime.com%2Fja%2Farea%2Fjp%2Fguide%2FNTJmat1113)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix  #109544

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+travel.navitime.com##.app-presents-frame
+travel.navitime.com##.floating-button__map
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:26]: It looks like a functional button - @Yuki2718 please take a look.

Comment (Yuki2718) [reply AnnoyancesFilter/sections/mobile-app.txt:26]: Yeah, this button shows the points of interests shown in the article in map so will be useful to user.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
~~Add `travel.navitime.com##.app-banner-header` too.~~ Move an exisiting rule `navitime.com##.app-banner-header` to the bottom section i.e. just above
```
!+ NOT_OPTIMIZED
hinative.com###fullscreen_app_recommend
```
as of writing this now and insert `!+ NOT_OPTIMIZED` just above the rule, i.e.
```
!+ NOT_OPTIMIZED
navitime.com##.app-banner-header
!+ NOT_OPTIMIZED
hinative.com###fullscreen_app_recommend
```

**Comment by SKEIDs** [comment]
@Yuki2718 
Done

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**